render_templates.sh

```diff
@@ -0,0 +1,54 @@
+#
+# MIT License
+#
+# (C) Copyright 2022 Hewlett Packard Enterprise Development LP
+#
+# Permission is hereby granted, free of charge, to any person obtaining a
+# copy of this software and associated documentation files (the "Software"),
+# to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense,
+# and/or sell copies of the Software, and to permit persons to whom the
+# Software is furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included
+# in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+#
+# Render templates from component_versions.sh
+
+ROOTDIR="$(dirname "${BASH_SOURCE[0]}")"
+
+# Set to "unstable" for artifacts from main branch, "stable" for artifacts from
+# release branch.
+SAT_REPO_TYPE="unstable"
```

Comment:
I realize this is just copied out from `release.sh`, but it might be nice while we're looking at this to automate this bit, for example by wildcard globbing the branch against `release/*` or something.

Reply:
Agreed, this line particular will need special attention if we backport this change to the release branch.

Reply:
> for example by wildcard globbing the branch against release/* or something.

I think this would end up giving us problems when creating builds based off release branches since this value is used to find the locations of artifacts. We would have to do something like figuring out of we were based off a release branch or an unstable branch. I'm going to see if we've discussed this before because this feels familiar.

Reply:
We have discussed it before, and for the reason you mention (getting it to work correctly on branches created from release branches vs. main), we just decided to hard-code it to "unstable" and update it when creating release branches.